Framework/PythonInterface/mantid/dataobjects/src/Exports/GroupingWorkspace.cpp

```diff
@@ -17,7 +17,10 @@ using namespace boost::python;
 GET_POINTER_SPECIALIZATION(GroupingWorkspace)
 
 void export_GroupingWorkspace() {
-  class_<GroupingWorkspace, bases<SpecialWorkspace2D>, boost::noncopyable>("GroupingWorkspace", no_init);
+  class_<GroupingWorkspace, bases<SpecialWorkspace2D>, boost::noncopyable>("GroupingWorkspace", no_init)
+      .def("getTotalGroups", &GroupingWorkspace::getTotalGroups, (arg("self")))
```

Comment:
These should use `return_value_policy<VectorToNumpy>()`. I'm looking for a better example than [TimeSeriesProperty](https://github.com/mantidproject/mantid/blob/main/Framework/PythonInterface/mantid/kernel/src/Exports/TimeSeriesProperty.cpp#L93-L96)

Reply:
[DeltaEMode](https://github.com/mantidproject/mantid/blob/main/Framework/PythonInterface/mantid/kernel/src/Exports/DeltaEMode.cpp#L29-L30) is another option, but it is returning strings.